spec/generators/rolemodel/rspec_generator_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+require 'generators/rolemodel/testing/rspec/rspec_generator'
+
+RSpec.describe Rolemodel::Testing::RspecGenerator, type: :generator do
+  destination File.expand_path('tmp/', File.dirname(__FILE__))
+
+  before(:all) do
+    prepare_test_app
+    run_generator
```

Comment:
Try this:

```suggestion
    FileUtils.cd(destination_root) { run_generator }
```

I'm going to work on a PR to move these before/after blocks to the global RSpec config..  Unless you feel like doing that in this PR?  I would support that!

Reply:
I'll hold off on that. Mostly just want to get this generator working